raster/src/main/scala/geotrellis/raster/EuclideanDistanceTile.scala

```diff
@@ -0,0 +1,29 @@
+package geotrellis.raster
+
+import geotrellis.raster.rasterize.polygon._
+import geotrellis.vector._
+import geotrellis.vector.voronoi._
+import scala.math.sqrt
+
+object EuclideanDistanceTile {
+
+  def apply(pts: Array[Point], re: RasterExtent): Tile = {
+    val vor = new Voronoi(pts, re.extent)
+    val tile = DoubleArrayTile.empty(re.cols, re.rows)
+    
+    def fillFn(base: Point)(c: Int, r: Int): Unit = {
+      val (x,y) = re.gridToMap(c,r)
+      tile.setDouble(c,r,sqrt((x-base.x)*(x-base.x) + (y-base.y)*(y-base.y)))
```

Comment:
Formatting: spaces after comma. Spaces between operands. 
Everywhere else we give the argument the names `col` and `row`, should follow that pattern (and generally use the longer names for clarity, and to fit the rest of the library, e.g. "points" instead of "pts", "rasterExtent" instead of "re")